src/main/java/org/sagebionetworks/web/client/widget/entity/menu/v2/Action.java

```diff
@@ -61,4 +61,8 @@ public enum Action {
   CREATE_EXTERNAL_DOCKER_REPO,
   CREATE_TABLE_VERSION,
   SHOW_PROJECT_STATS,
+  REPORT_VIOLATION,
+  DOWNLOAD_FILE,
+  ADD_TO_DOWNLOAD_CART,
+  SHOW_PROGRAMMATIC_OPTIONS,
```

Comment:
These actions were moved into this menu for v3. They don't get configured/exposed in the existing implementation, so it shouldn't cause any issues.